File: src/main/java/io/github/support/terminal/application/domains/core/api/telegram/responses/SendMessageResponse.java
package io.github.support.terminal.application.domains.core.api.telegram.responses;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.github.support.terminal.application.domains.core.api.telegram.models.Message;
import lombok.Data;
import lombok.ToString;
import lombok.experimental.Accessors;

@Data
@ToString
@Accessors(chain = true)
public class SendMessageResponse {

    @JsonProperty("ok")
    private Boolean ok;

    @JsonProperty("result")
    private Message result;

}

File: src/main/kotlin/io/github/support/terminal/application/domains/integrations/telegram/responses/SendMessageResponse.kt
package io.github.support.terminal.application.domains.integrations.telegram.responses

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import io.github.support.terminal.application.domains.integrations.telegram.models.Message

@JsonIgnoreProperties(ignoreUnknown = true)
data class SendMessageResponse (
    @JsonProperty("ok") val ok: Boolean,
    @JsonProperty("result") val result: Message
)